blue_localization/blue_localization/localizer.py

```diff
@@ -141,16 +183,12 @@ def __init__(self, node_name: str) -> None:
             qos_profile_default,
         )
 
-    def publish(self, twist: TwistStamped | TwistWithCovarianceStamped) -> None:
-        """Publish a twist message to the ArduSub EKF.
-
-        Args:
-            twist: The state message to send.
-        """
-        if isinstance(twist, PoseStamped):
-            self.vision_speed_pub.publish(twist)
+    def publish(self) -> None:
+        """Publish a twist message to the ArduSub EKF."""
+        if isinstance(self.state, PoseStamped):
```

Comment:
If we are checking the types here, then it would be good to throw an exception when an unsupported type is provided